x-pack/plugins/apm/public/components/app/error_group_overview/List/__snapshots__/List.test.tsx.snap

```diff
@@ -1,353 +1,279 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ErrorGroupOverview -> List should render empty state 1`] = `
-<Memo(UnoptimizedManagedTable)
```

Comment:
[enzyme-to-json@3.6.0](https://github.com/adriantoine/enzyme-to-json/releases/tag/v3.6.0) fixed deep serialization of memoized components